### PR TITLE
Add quality to federated showcase

### PR DESCRIPTION
**Pull request by EvaGarciaAlvarez**

### CODE REVIEW

File: showcase/prototyping-research-questions.md [hunk 1]
```diff
@@ -52,20 +52,60 @@ Each Participant Node (i.e., institutions hosting or being able to require acces
...
+
+The data validation report contains a validation compliance table and a validation plot informing on the rationale of each validation rule, their name (label), the number of items or registries for which the rule is applied, the number of registries passing and/or failing the rule, the percentage of failed registries within a variable, the number and percentage of missing values (namely ‘NA’s values), and if the rule has triggered any errors or warnings within the analysis. Non-compliance with the CDM is also informed globally as the number of registries failing a validation rule. 
+
+This study is built upon the implementation of a causal model to evaluate the effectiveness of the primary vaccination for COVID-19 at the population level. The methodology to support the analysis required for causal inference, requires establishing  counterfactual risks, achieved in this instance by matching similar individuals who completed the primary vaccination with individuals that did not or partially complete a primary  vaccination schedule, based on a set of assumed measurable confounders.
```
Comment (hedi-ee) [inline showcase/prototyping-research-questions.md:84]:
```suggestion
This study is built upon the implementation of a causal model to evaluate the effectiveness of the primary vaccination for COVID-19 at the population level. The methodology to support the analysis required for causal inference, requires establishing counterfactual risks, achieved in this instance by matching similar individuals who completed the primary vaccination with individuals that did not or partially complete a primary vaccination schedule, based on a set of assumed measurable confounders. 
```

File: showcase/prototyping-research-questions.md [hunk 2]
```diff
@@ -52,20 +52,60 @@ Each Participant Node (i.e., institutions hosting or being able to require acces
...
+
+This study is built upon the implementation of a causal model to evaluate the effectiveness of the primary vaccination for COVID-19 at the population level. The methodology to support the analysis required for causal inference, requires establishing  counterfactual risks, achieved in this instance by matching similar individuals who completed the primary vaccination with individuals that did not or partially complete a primary  vaccination schedule, based on a set of assumed measurable confounders. 
+
+Missing values in those variables considered as potential confounders could difficult the matching process and could potentially affect the interpretation of the analyses. That is why, once the data quality and the data validation rules has been assessed, a more detailed analysis is continued on those variables required for the matching process, so considered ‘core’ variables, following a standardised decision tree guiding the imputation process for those variables (see Figure 4 below).
```
Comment (hedi-ee) [inline showcase/prototyping-research-questions.md:86]:
```suggestion
Missing values in those variables considered as potential confounders could difficult the matching process and could potentially affect the interpretation of the analyses. That is why, once the data quality and the data validation rules has been assessed, a more detailed analysis is continued on those variables required for the matching process, so considered ‘core’ variables, following a standardised decision tree guiding the imputation process for those variables (see Figure 4 below). Missing values are further categorised. If the probability of being missing is the same for all cases, then the data are said to be missing completely at random (MCAR). If the probability of being missing is the same only within groups defined by the observed data, then the data are missing at random (MAR). The situation where the likelihood of missing data is related to the value of the missing data itself refers to missing not at random (MNAR).  In cases where the probability of missing data varies and cannot be fully explained by the observed data, the data are considered non-MAR (non-missing at random), indicating that the missingness may depend on unobserved factors.
```

**Review by hedi-ee — COMMENTED** [review]
Few changes proposed

**Review by bedroesb — APPROVED** [review]
Looks good!

**Pull request merged**